modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets. 
```

Comment:
I believe instances of `worker node` should be changed to `compute node`, unless its in a code block or a parameter. There are a few throughout this PR so I wanted to mention it

Reply:
Edge worker nodes would be correct, based on already published content in the 4.12 release.

Example of 4.12 content migrated to 4.13: https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-localzone.html